dev/index.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 RoadieHQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createDevApp } from '@backstage/dev-utils';
+import { CatalogClient, catalogApiRef } from '@backstage/plugin-catalog';
+import { githubPullRequestsApiRef, GithubPullRequestsClient } from '../src/api';
+import { plugin } from '../src/plugin';
+
+createDevApp()
+  .registerPlugin(plugin)
+  .registerApiFactory({
+    deps: {},
+    factory: () => new GithubPullRequestsClient(),
+    implements: githubPullRequestsApiRef,
+  })
+  .registerApiFactory({
```

Comment:
I presume that `yarn start` in this repo will work for development as long as the catalog is running on port 7000?

Reply:
Yes, I couldn't get the port from config in any way using `loadConfig` method but it didn't succeed. However when we are using this plugin as a separate repo we don't even have access to app-config.yaml, so maybe it's good that way